README.md

```diff
@@ -62,7 +62,23 @@
     - [Customize a Motion to fit a robot's action space.](#customize-a-motion-to-fit-a-robots-action-space)
     - [Run a robotics transformer model on a robot.](#run-a-robotics-transformer-model-on-a-robot)
     - [Notebooks](#notebooks)
-  - [The Sample Class and Building Blocks](#the-sample-class-and-building-blocks)
+  - [The Sample Class](#the-sample-class)
```

Comment:
This is too long. Can you just make it just sample class and building blocks and remove the sub-items?

Reply:
The feedback I got is that the table of contents didn't show enough of the documentation that was available.